.github/workflows/ci-cd.yaml

```diff
@@ -0,0 +1,29 @@
+name: ci-cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: ./bin/hermit env -r >> $GITHUB_ENV
+    - run: make
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [ci]
+    if: github.event_name == 'push' # only run on push to master
+    steps:
+    - uses: actions/checkout@v2
+    - run: git fetch --prune --unshallow --tags
+    - run: ./bin/hermit env -r >> $GITHUB_ENV
+    - run: make release
+    - if: ${{ failure() }}
```

Comment:
should this failure case also be duplicated into the `ci` job above?

Reply:
hmm, yes you are right.

Reply:
I'm almost inclined not to add it for the release.